app/components/ProgressStepIndicator.tsx

```diff
@@ -39,15 +58,21 @@ const NumberedCircle: React.FunctionComponent<NumberCircleProps> = ({
 
 export const ProgressStepIndicator: React.FunctionComponent<Props> = ({
   steps,
-  title
+  title = null,
+  ariaLabel = null
 }) => {
+  const totalCompleted = steps.reduce(function sumCompleted(acc, curr) {
+    return curr.completed ? acc + 1 : acc;
+  }, 0);
```

Comment:
this may be more readable as
`steps.filter(s => s.completed).length`
but works well as it is, there are a million different ways of writing map/filter/reduce operations